manager/assets/modext/widgets/element/modx.panel.tv.renders.js

```diff
@@ -59,7 +59,7 @@ Ext.reg('modx-panel-tv-image',MODx.panel.ImageTV);
 
 MODx.panel.FileTV = function(config) {
     config = config || {};
-    config.filemanager_url = MODx.config.filemanager_url;
+    config.filemanager_url = '';
```

Comment:
Does it make sense to find usage of this config param and remove it too?

Reply:
In my opinion, this setting is not used anywhere (the search finds only in this file), apparently it is more correct to delete `config.filemanager_url`

Reply:
What I am talking about :)